### PR TITLE
README: Add sentence to explain what project is

### PR DESCRIPTION
**Pull request by rjmunro**
Copied from http://ardupilot.org/about page.

I randomly came across this project on GitHub, and wondered what it was. The README just listed names like ArduPlane, ArduCopter, ArduRover, without saying what these mean.

### CODE REVIEW

**Comment by khancyr** [comment]
Hello,

Thanks, I like that.
Just about the commit message it miss "README: " before Add sentence to explain what project is

**Comment by rjmunro** [comment]
@khancyr I've adjusted the commit message.

**Comment by khancyr** [comment]
thanks!

File: README.md [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 [![Autotest Status](http://autotest.ardupilot.org/autotest-badge.svg)](http://autotest.ardupilot.org/)
 
+Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
+been developed over 5+ years by a team of diverse professional engineers and computer scientists. It is the
```
Comment (amilcarlucas) [inline README.md]: Instead of the 5+ years time span, just mention that it is "under development since 2010"

Comment (rjmunro) [reply README.md]: @amilcarlucas Would it make sense to also say that at http://ardupilot.org/about?

File: README.md [hunk 2]
```diff
@@ -8,6 +8,12 @@
 
 [![Autotest Status](http://autotest.ardupilot.org/autotest-badge.svg)](http://autotest.ardupilot.org/)
 
+Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
+been under development since 2010 by a team of diverse professional engineers and computer scientists. It is
+the only autopilot software capable of controlling any vehicle system imaginable, from conventional
```
Comment (tridge) [inline README.md]: i can imagine vehicles we don't support, there is a really cool combined submarine plane that we can't support. I think we need to tone this down in the place this came from too

Comment (tridge) [reply README.md]: add word 'almost' ?

**Comment by peterbarker** [comment]
I've added the word "almost" to this PR.

**Comment by peterbarker** [comment]
(the problem is also being fixed elsewhere, not original submitter's fault!)

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**